terraform/environments/analytical-platform-compute/helm-charts-actions-runners.tf

```diff
@@ -48,6 +48,31 @@ resource "helm_release" "actions_runner_mojas_airflow_create_a_pipeline" {
     )
   ]
 }
+/* CT-Tact-List */
+
+resource "helm_release" "actions_runner_ct_tact_list" {
+  count = terraform.workspace == "analytical-platform-compute-production" ? 1 : 0
+
+  /* https://github.com/ministryofjustice/analytical-platform-actions-runner */
+  name       = "actions-runner-ct-tact-list"
+  repository = "oci://ghcr.io/ministryofjustice/analytical-platform-charts"
+  version    = "2.320.0-4"
+  chart      = "actions-runner"
+  namespace  = kubernetes_namespace.actions_runners[0].metadata[0].name
+  values = [
+    templatefile(
+      "${path.module}/src/helm/values/actions-runners/ct-tact-list/values.yml.tftpl",
+      {
+        github_app_application_id  = jsondecode(data.aws_secretsmanager_secret_version.actions_runners_token_apc_self_hosted_runners_github_app[0].secret_string)["app_id"]
+        github_app_installation_id = jsondecode(data.aws_secretsmanager_secret_version.actions_runners_token_apc_self_hosted_runners_github_app[0].secret_string)["installation_id"]
+        github_organisation        = "moj-analytical-services"
+        github_repository          = "ct-tact-list"
+        github_runner_labels       = "analytical-platform"
+        eks_role_arn               = "arn:aws:iam::${local.environment_management.account_ids["analytical-platform-data-production"]}:role/data-iam-creator"
```

Comment:
This doesn't feel like the right role. Why data-iam-creator?
Also -- can you have another look at the naming?